.circleci/config.yml

```diff
@@ -1,3 +1,5 @@
+orbs:
+  go: circleci/go@1.5.0
```

Comment:
could you check again version of go? we're using 1.7

Reply:
yes - but I had to keep 1.5.0 for go circle ci executor as some libraries versions were mismatching. The circleci orbs for 1.7.0 doesn't seem to be working fine right now. 
Besides, only the testing code is gonna run with this executor and not our actual code. 
I'll keep an eye on circleci go executor updates and update in when it makes sense